hedera-node/hedera-app/src/test/java/com/hedera/node/app/throttle/impl/NetworkUtilizationManagerImplTest.java

```diff
@@ -110,4 +114,13 @@ void verifyTrackFeePayments() {
         verify(throttleAccumulator).checkAndEnforceThrottle(expectedTxnToBeChargedFor, consensusNow, state);
         verify(congestionMultipliers).updateMultiplier(consensusNow);
     }
+
+    @Test
+    void verifyShouldThrottlePostHandle() {
+        // when
+        subject.shouldThrottlePostHandle(transactionInfo, streamBuilder, consensusNow);
+
+        // then
+        verify(throttleAccumulator).checkAndEnforcePostHandleThrottle(transactionInfo, streamBuilder, consensusNow);
```

Comment:
Does this test ever fail? I don't see `transactionInfo` mocked anywhere, which suggests to me that this test isn't actually verifying anything about the transaction passed in

Reply:
Seems like a fairly useless test unless the `NetworkUtilizationManager.shouldThrottleByOpsDuration` is ever rewritten to not call `throttleAccumulator.checkAndEnforceOpsDurationThrottle`.  I can remove it if you want